### ISSUE TITLE
Crosswalk Android

### ISSUE THREAD

**Issue by matikbird** [nodekit-io/nodekit#8]
Hi!
For what I've read, this plugin it's amazing, but..

How can I set up a Cordova - Crosswalk webview with actually Node.js (like Electron, or NW.js) in Android, build the .apk and upload it to Google Play?
Are you telling me that I can run node, and all node modules like electron in a Crosswalk Web View and deploy it to Android?

If that ever happened, I have to see it. And reproduce it. Please.

**Comment by guycreate** [comment]
Yes, this is the goal for Android.  Will support both Crosswalk and native WebView on Android >4.4  (the latter to avoid the 20Mb overhead of Crosswalk, the former for those who need the advanced WebGL and more modern Chromium).      Note that only Node modules that don't have native GYP compilations will work, but this is a huge part of the Node.js ecosystem (just not "all node modules") and even those that do use native often don't really need to.  

Check back in 6 weeks for an Android sample that you can reproduce.
